### PR TITLE
Feature: As a user viewing a Participant's appointments, I should see…

### PR DESCRIPTION
**Pull request by goleksiak**
… them ordered by completed_date

### CODE REVIEW

File: app/assets/javascripts/participants.js.coffee [hunk 1]
```diff
@@ -60,3 +60,8 @@ $ ->
   $("#participants-list-table").bootstrapTable 'refresh', {silent: true}
   $("#participants-tracker-table").bootstrapTable 'refresh', {silent: true}
   $("#participant-info").bootstrapTable 'refresh', {silent: true}
+
+(exports ? this).visitSorter = (a, b) ->
```
Comment (kyle-glick) [inline app/assets/javascripts/participants.js.coffee]: This method name is a little ambiguous. It looks like a comparison function rather than a sorter.

Comment (goleksiak) [inline app/assets/javascripts/participants.js.coffee]: Got any suggestions for a different name?

Comment (kyle-glick) [inline app/assets/javascripts/participants.js.coffee]: I would call it compareVisit and make it return 1 if a > b and -1 if a < b

Comment (ctt3) [inline app/assets/javascripts/participants.js.coffee]: I disagree, as it is used in the context of sorting visits, I think visitSorter is just dandy

**Pull request merged**